core/extension/gdextension.h

```diff
@@ -155,6 +157,8 @@ class GDExtension : public Resource {
 
 	void track_instance_binding(Object *p_object);
 	void untrack_instance_binding(Object *p_object);
+
+	PackedStringArray get_used_classes() const;
```

Comment:
```suggestion
	PackedStringArray get_classes_used() const;
```

Better to stick with the naming used in the codebase.

Reply:
Ah, good catch, thanks! Updated all the names in my latest PR